xcode/Subconscious/Shared/Components/Common/Story/StoryUserView.swift

```diff
@@ -116,6 +129,10 @@ struct StoryUserView: View {
         }
         .contentShape(.interaction, RectangleCroppedTopRightCorner())
         .onTapGesture {
+            guard story.resolutionStatus == .resolved else {
+                return
+            }
```

Comment:
Preference for this no-op to belong to the update function, rather than the view. When actions handle their own no-op based on model state, it's more easily testable. It also makes sending the action foolproof.

Non-blocking.

Reply:
I introduced a new action `requestNavigateToProfile` which can result in `pushDetail` if resolved or `failPushDetail` if not. This logs a debug message and does nothing else atm.